File: Second Year (2013-2014)/OMGL7/TP2 - Website in PHP/musiconline/musiconline/db/Categorie.php
<?php
/* Authors: ANDRIAMILANTO Tompoariniaina && CANNO Camille
* Groupe: B1
* File: Categorie.php
*/

class Categorie implements DataObject{
   private $IdCategorie;
   private $LibelleCategorie;

   public function  __construct() { }

   public function init($IdCategorie,$LibelleCategorie){
      $this->IdCategorie=$IdCategorie;
      $this->LibelleCategorie=$LibelleCategorie;
   }


   public function getIDCategorie(){
     return $this->IdCategorie;
   }

   public function setIDCategorie($IdCategorie){
     $this->IdCategorie = $IdCategorie;
   }


   public function getLIbelleCategorie(){
     return $this->LibelleCategorie;
   }

   public function setLIbelleCategorie($LibelleCategorie){
     $this->LibelleCategorie = $LibelleCategorie;
   }


}
?>

File: Second Year (2013-2014)/OMGL7/TP3 - Website in JSP/src/db/Categorie.java
package db;
/**
*@author Andriamilanto Tompoariniaina - Canno Camille - Info 2 B
*/

/**
*Classe pour les objets Categorie
*/
public class Categorie {
//Attributs
  private int idCategorie;
  private String libelleCategorie;

//Methodes
  /**
  *Constructeur
  */
  public Categorie() {}

  /**
  *Methode pour initialiser les attributs
  */
  public void init(int idC, String lib) {
    this.idCategorie = idC;
    this.libelleCategorie = lib;
  }

//Accesseurs
  public void setIdCategorie(int idC) {
    this.idCategorie = idC;
  }

  public int getIdCategorie() {
    return this.idCategorie;
  }

  public void setLibelleCategorie(String libC) {
    this.libelleCategorie = libC;
  }

  public String getLibelleCategorie() {
    return this.libelleCategorie;
  }
}
